src/Serilog.Ui.Web/Extensions/SerilogUiOptionBuilderExtensions.cs

```diff
@@ -34,5 +35,32 @@ public static SerilogUiOptionsBuilder EnableAuthorization(this SerilogUiOptionsB
 
             return optionsBuilder;
         }
+
+        /// <summary>
+        /// Injects additional CSS stylesheets into the index.html page
+        /// </summary>
+        /// <param name="options"></param>
+        /// <param name="path">A path to the stylesheet - i.e. the link "href" attribute</param>
+        /// <param name="media">The target media - i.e. the link "media" attribute</param>
+        public static void InjectStylesheet(this UiOptions options, string path, string media = "screen")
+        {
+            var builder = new StringBuilder(options.HeadContent);
+            builder.AppendLine($"<link href='{path}' rel='stylesheet' media='{media}' type='text/css' />");
+            options.HeadContent = builder.ToString();
+        }
+
+        /// <summary>
+        /// Injects additional Javascript files into the index.html page
+        /// </summary>
+        /// <param name="options"></param>
+        /// <param name="path">A path to the javascript - i.e. the script "src" attribute</param>
+        /// <param name="type">The script type - i.e. the script "type" attribute</param>
+        public static void InjectJavascript(this UiOptions options, string path, string type = "text/javascript")
```

Comment:
```suggestion
        public static UiOptions InjectJavascript(this UiOptions options, string path, string type = "text/javascript")
```
Would you like the idea of returning UiOptions itself, to have a nicer fluent registration of the injections? An alternative could be to provide overloads that accept `params string[] `of paths...

Reply:
Yeah i was playing with the same idea, but you're accessing the `UIOptions` in a `Action` already so there is no real need to use a fluent API:

``` csharp
app.UseSerilogUi(x =>
{
    x.RoutePrefix = "serilog-ui";
    x.HomeUrl = "/#Test";
    x.InjectJavascript("/js/serilog-ui/custom.js")
});
```

On the other hand i don't see any downfalls so i'll add it in.